### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

- English sentences exist in this PR. Please leave them as they are. (They will be translated next time.)

- Some URLs of the docs do not have /ja. Please leave them as they are. (They will be handled on the next translation management system.)

### CODE REVIEW

**Review by catherinepope — APPROVED** [review]
😸

File: jekyll/_cci2_ja/creating-orbs.md [hunk 1]
```diff
@@ -18,119 +18,66 @@ version:
...
 
-### Orb のパブリッシュプロセス
-{: #orb-publishing-process }
+以下では、Orb の新しいセマンティックリリースを公開する方法について説明します。
 
-ここでは、Orb 開発キットについて掘り下げ、Orb のパブリッシュに関係するコンポーネントについて説明します。
+`circleci orb init` コマンドでサンプルの Orb プロジェクトを生成すると、自動的に `alpha` ブランチに移行されます。 ブランチ名は任意で構いませんが、デフォルト以外のブランチで変更することをお勧めします。
 
-[circleci orb init]({{site.baseurl}}/2.0/orb-author/#getting-started) コマンドは、最適な Orb 開発パイプラインで設計された定義済みの CircleCI 設定ファイルを含む [Orb テンプレートリポジトリ](https://github.com/CircleCI-Public/Orb-Project-Template)を、Orb 用にクローンします。
+変更したら、コミットしブランチにプッシュします。 コミットメッセージには、[Conventional Commits](https://www.conventionalcommits.org/) を使用することを強くお勧めします。
```
Comment (nanophate) [inline jekyll/_cci2_ja/creating-orbs.md:37]: 📝 https://www.conventionalcommits.org/ja
  It looks like they have Japanese page as well. Might be better to reference for usability.

File: jekyll/_cci2_ja/creating-orbs.md [hunk 2]
```diff
@@ -18,119 +18,66 @@ version:
...
+1. **新しいプルリクエストをデフォルトのブランチに作成します**。`./circleci` ディレクトリに含まれる`config.yml` ファイルと `test-deploy.yml` ファイルは、CircleCI Web アプリで Orb の変更を自動的に
+リント、[シェルチェック]({{site.baseurl}}/ja/2.0/testing-orbs/#shellcheck)、[レビュー]({{site.baseurl}}/ja/2.0/testing-orbs/#review)、[テスト]({{site.baseurl}}/ja/2.0/testing-orbs/#integration-testing)します。
+   
+1. **すべてのテストが成功したか確認してください。**<br/>
+テスト結果は、GitHub 上のプルリクエストで直接確認できます。 また、CircleCI Web アプリではパイプライン全体に対する詳細な結果を確認できます。 ワークフローが 2 つあり、`lint-pack` が先に実行され、リント、シェルチェック、レビューを含まれており、2 つ目のワークフローでテストするよう開発版をパブリッシュします。 この `test-deploy` には結合テストが含まれており、準備が整い次第 Orb の安定版をパブリッシュできます。 ![プルリクエストに対して GitHub Checks API から返された Orb のテスト結果レポート]({{site.baseurl}}/assets/img/docs/orbtools-11-checks.png)
 
-#### test-pack
-{: #test-pack }
+1. **"スカッシュ" マージ** <br/>
+変更が完了したら、[Conventional Commit メッセージ](https://www.conventionalcommits.org/) を使って変更を一つのコミットに "[スカッシュマージ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits)" することをお勧めします（必須ではありません）。
```
Comment (nanophate) [inline jekyll/_cci2_ja/creating-orbs.md:48]: 📝 https://www.conventionalcommits.org/ja

File: jekyll/_cci2_ja/creating-orbs.md [hunk 3]
```diff
@@ -18,119 +18,66 @@ version:
...
-    | パブリッシュされる開発タグ                                          | 説明                                         |
-    | ------------------------------------------------------ | ------------------------------------------ |
-    | `<namespace>/<orb>@dev:<branch>`     | ブランチ名にリンクされる開発タグです。 設定ファイルのテストを行う場合に使用します。 |
-    | `<namespace>/<orb>@dev:${CIRCLE_SHA1:0:7}` | この SHA に固有の開発タグです。 次のワークフローで使用します。         |
-    {: class="table table-striped"}
+GitHub のドキュメントに従って新しいリリースを作成します。
 
-テストジョブの詳細については、[Orb のテスト手法]({{site.baseurl}}/2.0/testing-orbs)を参照してください。
+   1. タグの選択を求められたら、今回のリリースでパブリッシュされる新しいタグを指定します。 Orb の現在のバージョンを [Orb レジストリ](https://circleci.com/developer/ja/orbs)で再度確認します。 どのような変更を行っているかわからない場合は、この後修正することができます。 タグの形式が `vX.Y.Z` であることを確認してください。
+   2. [+ Auto-generate release notes] ボタンをクリックします。 変更点のまとめが作成されます。これには前回のリリース以降にマージされたすべてのプルリクエストが含まれます。 [Conventional Commit メッセージ](https://www.conventionalcommits.org/) を使用したことがあれば、セマンティックインクリメントの変更タイプはコミットメッセージにより簡単に分かります。 たとえば、前に "fix:" がついた 2 つのコミットがある場合、セマンティックバージョンが "patch" レベルでインクリメントすることが予想されます。
```
Comment (nanophate) [inline jekyll/_cci2_ja/creating-orbs.md:65]: 📝 https://www.conventionalcommits.org/ja

**Review by nanophate — APPROVED** [review]
I noticed some external link had Japanese translation. Other than that it looks great! 🎉

File: jekyll/_cci2_ja/creating-orbs.md (hunk 1 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/creating-orbs.md]: Thank you! I've added /ja.

**Pull request merged**